pom.xml

```diff
@@ -0,0 +1,161 @@
+    or deviating elements (inconsistencies). An element can be any representable item of the model, like a component
+    or a relation. To do so, we first create trace links and then make use of them and other information to identify
+    inconsistencies. ArDoCo is actively developed by researchers of the Modelling for Continuous Software Engineering
+    (MCSE) group of KASTEL - Institute of Information Security and Dependability at the KIT. This work was supported by
+    funding from the topic Engineering Secure Systems of the Helmholtz Association (HGF) and by KASTEL Security Research
+    Labs (46.23.01).</description>
+
+  <licenses>
+    <license>
+      <name>MIT License</name>
+      <url>http://www.opensource.org/licenses/mit-license.php</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>Gram21</id>
+      <name>Jan Keim</name>
+      <email>jan.keim@kit.edu</email>
+      <url>https://mcse.kastel.kit.edu/staff_Keim_Jan.php</url>
+      <organization>KASTEL</organization>
+      <organizationUrl>https://mcse.kastel.kit.edu/</organizationUrl>
+      <timezone>GMT+1</timezone>
+    </developer>
+  </developers>
+  <scm>
+    <connection>scm:git:git://github.com/ArDoCo/Core.git</connection>
+    <developerConnection>scm:git:ssh://github.com:ArDoCo/GUI.git</developerConnection>
+    <url>http://github.com/ArDoCo/GUI/tree/main</url>
+  </scm>
+  <issueManagement>
+    <system>GitHub Issues</system>
+    <url>https://github.com/ArDoCo/GUI/issues</url>
+  </issueManagement>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>17</java.version>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <maven.compiler.release>17</maven.compiler.release>
+    <javafx.version>19</javafx.version>
+    <javafx-maven-plugin.version>0.0.6</javafx-maven-plugin.version>
+    <sonar.organization>ardoco</sonar.organization>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar.projectKey>ArDoCo_GUI</sonar.projectKey>
+    <sonar.coverage.jacoco.xmlReportPaths>target/site/jacoco-aggregate/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.github.ardoco.core</groupId>
+      <artifactId>pipeline</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.openjfx</groupId>
+      <artifactId>javafx-controls</artifactId>
+      <version>${javafx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.openjfx</groupId>
+      <artifactId>javafx-fxml</artifactId>
+      <version>${javafx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.testfx</groupId>
+      <artifactId>testfx-core</artifactId>
+      <version>4.0.16-alpha</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
```

Comment:
check. shouldn't be needed

Reply:
I think it might be actually needed here. The parents only include it in the `dependencyManagement` section and the dependency tree by maven only shows junit if we include it here.